modules/install-acs-operator.adoc

```diff
@@ -31,6 +31,14 @@ If you choose automatic updates, when a new version of the Operator is available
 +
 If you choose manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to update the Operator to the new version.
 
```

Comment:
I am not a big expert in AsciiDoc, but I think you may need to replace this empty line with the `+` sign in order to make the `[IMPORTANT]` fragment render at the same level of indentation as the sublist item and not break the outer list numbering.
Check how it currently renders in https://github.com/jschnath/openshift-docs/blob/rhacs-docs/modules/install-acs-operator.adoc

![image](https://user-images.githubusercontent.com/537715/202541027-3737395e-a287-41fe-a6d8-167342098a8e.png)

```suggestion
+
```

If this doesn't work, I'd suggest you to check the official docs how to make that look nice.

Reply:
This should work to keep the note in line with the previous step.

Reply:
I personally prefer not to indent banner blocks, but I'll do it if that's our style.

Reply:
The numbering is now consistent.
![image](https://user-images.githubusercontent.com/537715/202672461-98b3645c-3040-4326-ba75-4b0eb1302548.png)